src/mmw/js/src/draw/templates/delineationOptions.html

```diff
@@ -12,8 +12,14 @@
   </button>
   <ul class="dropdown-menu menu-left" role="menu">
     <li role="presentation">
-      <a role="menuitem" tabindex="-1" href="#" data-shape-type="stream" data-snapping-on="true">
-        <span>Stream Network</span> <i class="split fa fa-question-circle"
+      <a role="menuitem" tabindex="-1" href="#" data-shape-type="stream" data-snapping-on="true" data-data-source={{DRB}}>
+        <span>DRB Stream Network</span> <i class="split fa fa-question-circle"
+          data-content="This automatically snaps to the nearest point on a perennial stream and calculates the watershed boundaries for the stream and its tributaries upstream of the selected point."></i>
```

Comment:
We'll need to differentiate the text here between DRB and NHD Region 2, but let's leave it as is and ask the client for input so we don't have to update it twice.